src/cpp/src/whisper_pipeline_static.cpp

```diff
@@ -340,16 +542,57 @@ WhisperPipeline::StaticWhisperPipeline::StaticWhisperPipeline(const std::filesys
     auto decoder_model = core.read_model(models_path / "openvino_decoder_model.xml");
     auto decoder_with_past_model = core.read_model(models_path / "openvino_decoder_with_past_model.xml");
 
+    add_attention_mask_input_for_decoder(decoder_model);
```

Comment:
Does it make sense to move `add_attention_mask_input`, `reshape_to_static`, `preprocess_**` functions to a separate file under whisper folder?

Reply:
I don't think so, they're relevant only for static shape implementation